.vscodeignore

```diff
@@ -0,0 +1,18 @@
+__mocks__/

```

Comment:
gotta check that we don't package unwanted files

Reply:
looks appropriate in `~/.vscode/extensions/yocto-project.yocto-bitbake-2.4.0`. Could you also take a look and see if there's anything that should be removed? Maybe server node_modules?

Reply:
Perfect, no need to address package size.